client/app/dbaas/logs/detail/streams/alerts/home/alerts-home.controller.js

```diff
@@ -1,9 +1,112 @@
-class LogsStreamsAlertsHomeCtrl {
-    constructor ($stateParams) {
+class AlertsHomeCtrl {
+    constructor ($state, $stateParams, $translate, CloudMessage, ControllerHelper, LogsStreamsService, StreamsAlertsConstant, StreamsAlertsService) {
+        this.$state = $state;
         this.$stateParams = $stateParams;
+        this.$translate = $translate;
+        this.CloudMessage = CloudMessage;
+        this.ControllerHelper = ControllerHelper;
+        this.LogsStreamsService = LogsStreamsService;
+        this.StreamsAlertsConstant = StreamsAlertsConstant;
+        this.StreamsAlertsService = StreamsAlertsService;
+
         this.serviceName = this.$stateParams.serviceName;
-        this.stream = this.$stateParams.streamId;
+        this.streamId = this.$stateParams.streamId;
+        this._initLoaders();
+    }
+
+    $onInit () {
+        this.runLoaders();
+    }
+
+    /**
+     * Runs all the loaded to fetch data from APIs
+     *
+     * @memberof AlertsHomeCtrl
+     */
+    runLoaders () {
```

Comment:
_runLoaders

Reply:
That's if you want to be consequent with the rest of the file.  I won't decline your PR for that

Reply:
This is done :)